integration-cli/docker_cli_run_test.go

```diff
@@ -1331,7 +1331,7 @@ func (s *DockerSuite) TestRunResolvconfUpdate(c *check.C) {
 	}
 
 	if bytes.Equal(containerResolv, resolvConfSystem) {
-		c.Fatalf("Restarting  a container after container updated resolv.conf should not pick up host changes; expected %q, got %q", string(containerResolv), string(resolvConfSystem))
+		c.Fatalf("Restarting  a container after container updated resolv.conf should not pick up host changes; expected %q\nsearch mylittlepony.com\n, got %q", string(resolvConfSystem), string(containerResolv))
```

Comment:
I agree that `expected` and `obtained` should switch sides for clarity when debugging a failure.

I think this check is a little weird. I'm not sure inserting '\nsearch mylittlepony.com\n' directly after is the right approach.

I would maybe rewrite it to check that `containerResolv` contains the `search` line.